metrics-core/src/main/java/software/amazon/swage/metrics/record/cloudwatch/CloudWatchRecorder.java

```diff
@@ -153,6 +153,12 @@ public Builder scheduledExecutorService(ScheduledExecutorService scheduledExecut
         }
 
         public CloudWatchRecorder build() {
+            if (scheduledExecutorService == null) {
```

Comment:
Nitpick.
The commit message doesn't say why the executor was changed as part of this, nor does the code. We should either update the commit message or move the executor back to being defaulted.

Reply:
Fixed: https://github.com/awslabs/swage/pull/29/commits